testgrid/conformance/README.md

```diff
@@ -0,0 +1,38 @@
+# Display Conformance Tests with Testgrid
+
+This directory contains tooling for displaying [kubernetes conformance test](https://github.com/cncf/k8s-conformance) results on [testgrid](../README.md).
+
+
+## Using
```

Comment:
Would it make more sense to structure this as choose a strategy:

1) Follow the official guide
2) Use kubetest
3) Use sonobuouy

And then give people step by step instructions? While this duplicates stuff like creating a GCS bucket, it makes for an easier read. Aka we don't have (and now if you did X do Y or if you did P do Q)

Reply:
hmm. maybe, everything is the same except the log file locations [and the instructions for running the tests].